test/livebook/live_markdown/markdown_helpers_test.exs

```diff
@@ -108,6 +190,42 @@ defmodule Livebook.LiveMarkdown.MarkdownHelpersTest do
       assert markdown == reformat(markdown)
     end
 
+    test "code block with fences inside it" do
+      markdown = """
+      ~~~~elixir
+      before
+
+      ```
+      _inside_
+      ```
+
+      after
+      ~~~~\
+      """
+
+      assert markdown == reformat(markdown)
+    end
+
+    test "display math" do
+      markdown = """
+      $$
+      R_{ij}^{kl} = R_{ij} - \Gamma^k_{kl}
+      $$\
+      """
+
+      assert markdown == reformat(markdown)
+    end
+
+    test "alternate brackets for display math" do
+      markdown = """
+      \\[
+      R_{ij}^{kl} = R_{ij} - \Gamma^k_{kl}
+      \\]\
+      """
+
+      assert markdown == reformat(markdown)
+    end
```

Comment:
These alternate brackets are supported by KaTeX's auto-renderer by default, but the `\` disappear on formatting

Reply:
As said, I don't think we need to support these. The escaping with `\` is however an issue as the parser treats the content specially and discards it, so we don't really know it and would need to escape certain texts on our own :thinking: